app/assets/javascripts/admin/addon/controllers/modals/site-setting-default-categories.js

```diff
@@ -1,20 +1,9 @@
 import Controller from "@ember/controller";
 import ModalFunctionality from "discourse/mixins/modal-functionality";
+import ModalUpdateExistingUsers from "discourse/mixins/modal-update-existing-users";
 
-export default Controller.extend(ModalFunctionality, {
-  onShow() {
-    this.set("updateExistingUsers", null);
-  },
-
-  actions: {
-    updateExistingUsers() {
-      this.set("updateExistingUsers", true);
-      this.send("closeModal");
-    },
-
-    cancel() {
-      this.set("updateExistingUsers", false);
-      this.send("closeModal");
-    },
-  },
-});
+export default Controller.extend(
+  ModalFunctionality,
+  ModalUpdateExistingUsers,
+  {}
```

Comment:
I don't think this `{}` is required. The mixins should be enough. (same comment applies below.)